xDrip/Scenes/Settings/BaseSettings/Cells/BaseSettingsPickerExpandableTableViewCell.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  BaseSettingsPickerExpandableTableViewCell.swift
+//  xDrip
+//
+//  Created by Ivan Skoryk on 11.04.2020.
+//  Copyright © 2020 Faifly. All rights reserved.
+//
+
+import UIKit
+
+class BaseSettingsPickerExpandableTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak private var vertStackView: UIStackView!
+    @IBOutlet weak private var mainTextLabel: UILabel!
+    @IBOutlet weak private var detailLabel: UILabel!
+    
+    private var picker = UIView()
+    private var dataSource = [[String]]()
+    
+    func configure(mainText: String, detailText: String?, dataSource: [[String]], pickerView: UIView) {
```

Comment:
This cell shouldn't know about `dataSource`. Since you pass pickerView as a `UIView`, it's implementation should be hidden from the cell. Move it to presenter.